lib/rspec/core/lookup_set.rb

```diff
@@ -0,0 +1,32 @@
+module RSpec
+  module Core
+    # @private
+    class LookupSet
```

Comment:
I'll add a comment, I didn't want to shadow the standard lib name especially as it's not functionally complete